backend/app/routers/feeds.py

```diff
@@ -51,6 +51,7 @@ async def check_feed_listeners(
         feed_match = check_search_result(es_client, file_out, feed.search)
         if feed_match:
             for listener in feed.listeners:
+                # TODO is this necessary?
```

Comment:
yes, i think so. this will automatically trigger the listener/extractor on the newly uploaded file only if automatic is set to true, if it is false maybe the user only wants to send things to this extractor manually later.